sdk/search/azure-search-documents/tests/search_service_preparer.py

```diff
@@ -85,28 +83,33 @@ def _clean_up_indexers(endpoint, api_key):
             raise
 
 
-def _set_up_index(service_name, endpoint, api_key, schema, index_batch):
-    from azure.core.credentials import AzureKeyCredential
+def _set_up_index(service_name, endpoint, cred, schema, index_batch):
     from azure.search.documents import SearchClient
+    from azure.search.documents.indexes.models import SearchIndex
     from azure.search.documents._generated.models import IndexBatch
+    from azure.search.documents.indexes import SearchIndexClient
 
     schema = _load_schema(schema)
     index_batch = _load_batch(index_batch)
     if schema:
-        index_name = json.loads(schema)["name"]
-        response = requests.post(
-            SERVICE_URL_FMT.format(service_name, SEARCH_ENDPOINT_SUFFIX),
-            headers={"Content-Type": "application/json", "api-key": api_key},
-            data=schema,
-        )
-        if response.status_code != 201:
-            raise AzureTestError("Could not create a search index {}".format(response.status_code))
+        index_json = json.loads(schema)
+        index_name = index_json["name"]
+        index = SearchIndex.from_dict(index_json)
+        index_client = SearchIndexClient(endpoint, cred, retry_backoff_factor=60)
+        index_create = index_client.create_index(index)
+        # response = requests.post(
+        #     SERVICE_URL_FMT.format(service_name, SEARCH_ENDPOINT_SUFFIX),
+        #     headers={"Content-Type": "application/json", "Authorization": "Bearer {}".format(cred.get_token("https://search.azure.com/.default"))},
+        #     data=schema,
+        # )
+        # if response.status_code != 201:
+        #     raise AzureTestError("Could not create a search index {}".format(response.status_code))
```

Comment:
Seems like we should just delete this commented-out block, since the client is creating the index now (and appears to be working).